eventing/default-channels.md

```diff
@@ -0,0 +1,129 @@
+# Default Channels
+
+The default channel configuration allows channels to be created without
+specifying a provisioner. This leaves the selection of channel provisioner and
+properties up to the operator. The operator controls the default settings via a
+ConfigMap.
+
+## Creating a default channel
+
+To create a default channel, use the same procedure as for creating a normal
+channel, but leave the `spec.provisioner` property blank. The `spec` property
+must be provided, but should be empty.
+
+_The content of `spec.arguments` will be cleared for default channels._
+
+This is a valid default channel:
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Channel
+metadata:
+  name: default-channel
+  namespace: default
+spec: {}
+```
+
+When the above Channel is created, a mutating admission webhook sets
+`spec.provisioner` based on the default provisioner chosen by the operator.
+
+For example, if the default provisioner is named `default-provisioner`:
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Channel
+metadata:
+  name: default-channel
+  namespace: default
+spec:
+  provisioner:
+    apiversion: eventing.knative.dev/v1alpha1
+￼    kind: ClusterChannelProvisioner
+￼    name: default-provisioner
+```
+
+### Caveats
+
+#### Arguments cannot be specified by default channels
+
+If `spec.arguments` is set when creating a default channel, it will be cleared.
```

Comment:
Do you want to explain why this is true, assuming that it is intentional rather than a bug?

Reply:
Good point, this should be explained. IIUC it's a missing feature.